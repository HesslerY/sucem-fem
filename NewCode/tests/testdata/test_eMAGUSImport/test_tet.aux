BLOCK materials
2
SUBBLOCK
eps_r
1 0
material_label
1
material_type 
isotropic
ENDSUBBLOCK
SUBBLOCK
eps_r
1 0
material_label
2
material_type
isotropic
ENDSUBBLOCK
ENDBLOCK
