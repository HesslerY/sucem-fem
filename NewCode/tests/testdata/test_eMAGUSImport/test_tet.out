                    PROGRAM INFORMATION

 Program name:     FEMFEKO
 Description:      3 Dimensional ElectroMagnetic
                   Finite Element Method code
 Version number:   1.00--neilen-0.0        
 Revision number:  patch-14
 Last changed:     20050422
 Authors:          D.B.Davidson, M.M.Botha & R.H.Geschke
                   (Dept E&E Engr., Univ. of Stellenbosch, South Africa)
                   F.J.C.Meyer
                   (EMSS Pty Ltd, Stellenbosch, South Africa)
 Date of run(ymd): 20060421
 Time run started: 18h13


                    ASSOCIATED FILES

 Control data read from file: input.dat                                                                                                                       
 Mesh read from file:         test_tet.fek                                                                                                                    
 Output data written to file: +test_tet.out                                                                                                                   


                    ANALYSIS INFORMATION

 Elements used:          Constant Tangential/Linear Normal
 Elemental matrices:     Cubature used.
 Sparse matrix storage:  No
 Matrix solution scheme: LU factorization (direct)
 FE field averaging:     No


                    BOUNDARY CONDITIONS

      BC number        BC type           co-ordinates (x,y,z)


                    LIST OF NODES

                                LOCATION
      Node no.             X             Y             Z
          1     -0.50000E+00   0.50000E+00  -0.50000E+00
          2      0.50000E+00   0.50000E+00   0.50000E+00
          3      0.50000E+00  -0.50000E+00  -0.50000E+00
          4     -0.50000E+00  -0.50000E+00   0.50000E+00
          5      0.16667E+00   0.16667E+00  -0.16667E+00
          6     -0.16667E+00   0.16667E+00   0.16667E+00
          7     -0.16667E+00  -0.16667E+00  -0.16667E+00
          8      0.16667E+00  -0.16667E+00   0.16667E+00


                    LIST OF ELEMENTS

   Element no.          Type  Material no.        Node 1        Node 2        Node 3        Node 4
           1           CT/LN           1             1             4             6             7
           2           CT/LN           1             1             3             5             7
           3           CT/LN           1             3             4             7             8
           4           CT/LN           1             1             2             5             6
           5           CT/LN           1             2             3             5             8
           6           CT/LN           1             2             4             6             8
           7           CT/LN           1             4             6             7             8
           8           CT/LN           1             1             5             6             7
           9           CT/LN           1             3             5             7             8
          10           CT/LN           1             2             5             6             8
          11           CT/LN           1             5             6             7             8


                    MESH INFORMATION

 Number of nodes:              8
 Number of elements:          11
 Number of edges:             24
 Number of faces:             28
 Average edge length:   0.8797    


                    MATRIX INFORMATION

No. of degrees of freedom (free edges & faces):         6
Formulation used for S&T:                        Webb99
S and T matrices scaled by edge lengths:         No


                    LIST OF EDGES
       Edge no.      Node 1      Node 2    PEC   Port Port #    ABC  ABC #
            1           1           4       T       F     0        F     0
            2           1           6       T       F     0        F     0
            3           1           7       T       F     0        F     0
            4           4           6       T       F     0        F     0
            5           4           7       T       F     0        F     0
            6           6           7       F       F     0        F     0
            7           1           3       T       F     0        F     0
            8           1           5       T       F     0        F     0
            9           3           5       T       F     0        F     0
           10           3           7       T       F     0        F     0
           11           5           7       F       F     0        F     0
           12           3           4       T       F     0        F     0
           13           3           8       T       F     0        F     0
           14           4           8       T       F     0        F     0
           15           7           8       F       F     0        F     0
           16           1           2       T       F     0        F     0
           17           2           5       T       F     0        F     0
           18           2           6       T       F     0        F     0
           19           5           6       F       F     0        F     0
           20           2           3       T       F     0        F     0
           21           2           8       T       F     0        F     0
           22           5           8       F       F     0        F     0
           23           2           4       T       F     0        F     0
           24           6           8       F       F     0        F     0


                    LIST OF FACES
       Face no.      Node 1      Node 2      Node 3    PEC   Port Port #    ABC  ABC #
            1           1           4           6       T       F     0        F     0
            2           1           4           7       T       F     0        F     0
            3           1           6           7       F       F     0        F     0
            4           4           6           7       F       F     0        F     0
            5           1           3           5       T       F     0        F     0
            6           1           3           7       T       F     0        F     0
            7           1           5           7       F       F     0        F     0
            8           3           5           7       F       F     0        F     0
            9           3           4           7       T       F     0        F     0
           10           3           4           8       T       F     0        F     0
           11           3           7           8       F       F     0        F     0
           12           4           7           8       F       F     0        F     0
           13           1           2           5       T       F     0        F     0
           14           1           2           6       T       F     0        F     0
           15           1           5           6       F       F     0        F     0
           16           2           5           6       F       F     0        F     0
           17           2           3           5       T       F     0        F     0
           18           2           3           8       T       F     0        F     0
           19           2           5           8       F       F     0        F     0
           20           3           5           8       F       F     0        F     0
           21           2           4           6       T       F     0        F     0
           22           2           4           8       T       F     0        F     0
           23           2           6           8       F       F     0        F     0
           24           4           6           8       F       F     0        F     0
           25           6           7           8       F       F     0        F     0
           26           5           6           7       F       F     0        F     0
           27           5           7           8       F       F     0        F     0
           28           5           6           8       F       F     0        F     0


                    ELEMENT FACE INTERCONNECTIVITY DATA
        Element        FaceConnected toConnected to
            1           1               0        0        8        7         0        0        3        1
            1           2               0        0        8        7         0        0        3        1
            1           3               0        0        8        7         0        0        3        1
            1           4               0        0        8        7         0        0        3        1
            2           1               0        0        8        9         0        0        2        1
            2           2               0        0        8        9         0        0        2        1
            2           3               0        0        8        9         0        0        2        1
            2           4               0        0        8        9         0        0        2        1
            3           1               0        0        9        7         0        0        3        3
            3           2               0        0        9        7         0        0        3        3
            3           3               0        0        9        7         0        0        3        3
            3           4               0        0        9        7         0        0        3        3
            4           1               0        0        8       10         0        0        1        1
            4           2               0        0        8       10         0        0        1        1
            4           3               0        0        8       10         0        0        1        1
            4           4               0        0        8       10         0        0        1        1
            5           1               0        0       10        9         0        0        2        2
            5           2               0        0       10        9         0        0        2        2
            5           3               0        0       10        9         0        0        2        2
            5           4               0        0       10        9         0        0        2        2
            6           1               0        0       10        7         0        0        3        2
            6           2               0        0       10        7         0        0        3        2
            6           3               0        0       10        7         0        0        3        2
            6           4               0        0       10        7         0        0        3        2
            7           1               1        6        3       11         4        4        4        4
            7           2               1        6        3       11         4        4        4        4
            7           3               1        6        3       11         4        4        4        4
            7           4               1        6        3       11         4        4        4        4
            8           1               4        2        1       11         3        3        3        1
            8           2               4        2        1       11         3        3        3        1
            8           3               4        2        1       11         3        3        3        1
            8           4               4        2        1       11         3        3        3        1
            9           1               2        5        3       11         4        4        3        3
            9           2               2        5        3       11         4        4        3        3
            9           3               2        5        3       11         4        4        3        3
            9           4               2        5        3       11         4        4        3        3
           10           1               4        5        6       11         4        3        3        2
           10           2               4        5        6       11         4        3        3        2
           10           3               4        5        6       11         4        3        3        2
           10           4               4        5        6       11         4        3        3        2
           11           1               8       10        9        7         4        4        4        4
           11           2               8       10        9        7         4        4        4        4
           11           3               8       10        9        7         4        4        4        4
           11           4               8       10        9        7         4        4        4        4
